### PR TITLE
Formatting updates

### PR DESCRIPTION
**Pull request by ElleEss**
* Updates for consistency with Get started
* Updates for consistency across the docs
* Small formatting which can help user orientation - nothing major - made sure the in-doc references were OK

#### Tests
Improbadoc

#### Primary reviewers
@anne-edwards 
(Ernie and Rob (sick) FYI)
FYI also Ollie and Ben

See: https://docs-testing.improbable.io/unreal/alpha/index
(updated with Anne's suggestions at 13.34 on Friday 2019-05-24)

Screen Shot 2019-05-24 at 10 06 55 AM

### CODE REVIEW

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-authentication.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-## Step 2: Generate a development authentication token
+# Multiple deployments for session-based games
+## 2: Generate an authentication token
 
-**Note:** Development authentication tokens are for in-development testing only. They are for early-stage development so you don’t have to create your own game authentication service before testing your game clients. Do not include development authentication tokens in production projects. 
+>**Note:** This section describes generating and using development authentication tokens. These are for in-development testing only. They are for early-stage development so you don’t have to create your own game authentication service before testing your game clients. Do not include development authentication tokens in production projects. 
 
+### Step 1: Generate a toke
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-authentication.md]:
```suggestion
### Step 1: Generate a token
```

File: SpatialGDK/Documentation/content/pricing-and-support/pricing.md [hunk 2]
```diff
@@ -4,4 +4,7 @@ The GDK for Unreal uses SpatialOS core technology and cloud hosting. This means
 
 SpatialOS has a free tier and a paid tier, and you can use the GDK on either of these. For an introduction to the pricing model, see [Pricing introduction](https://docs.improbable.io/reference/latest/shared/pricing-and-support/pricing-intro).
 
-For more details, including information about what the free tier includes, an in-depth explanation of how we calculate prices, and examples of how much different types of games would cost, see [Pricing details](https://docs.improbable.io/reference/latest/shared/pricing-and-support/pricing-details).
\ No newline at end of file
+For more details, including information about what the free tier includes, an in-depth explanation of how we calculate prices, and examples of how much different types of games would cost, see [Pricing details](https://docs.improbable.io/reference/latest/shared/pricing-and-support/pricing-details).
+
+<br/>--------<br/>
+2019-05-20: Page added with editorial review
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/pricing-and-support/pricing.md]:
```suggestion
_2019-05-20: Page added with editorial review_
```

File: SpatialGDK/Documentation/content/pricing-and-support/support.md [hunk 3]
```diff
@@ -4,4 +4,7 @@ We provide free technical support to all users via our [forums](https://forums.i
 
 For more details, see [Support](https://docs.improbable.io/reference/latest/shared/pricing-and-support/support).
 
-We also offer additional paid tiers of support. For more details, [get in touch](https://improbable.io/contact-us).
\ No newline at end of file
+We also offer additional paid tiers of support. For more details, [get in touch](https://improbable.io/contact-us).
+
+<br/>--------<br/>
+2019-05-20: Page added with editorial review
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/pricing-and-support/support.md]:
```suggestion
_2019-05-20: Page added with editorial review_
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-configure.md [hunk 4]
```diff
@@ -50,15 +52,15 @@ Where `beta_nuts_double_379` is your SpatialOS cloud project name.
...
+### Step 3: Edit the config.json file
 
 Next, you configure the Deployment Manager. 
 
 In this example, you must tell the Deployment Manager:
 
 *  the type of client-worker it can deploy - `UnrealClient` for this example, but this can be any client type you have set up for your project
 * the number of deployments you want it to allow to run concurrently - 2 for this example
-* which worker assembly to use for the deployments - the name for which you created in step 4, above.
+* which worker assembly to use for the deployments - the name for this you created in step 2, above.
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-configure.md]:
```suggestion
* which worker assembly to use for the deployments - you created the name for this in step 2, above.
```
(I've suggested an update according to how I interpreted this)

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro.md [hunk 5]
```diff
@@ -1,21 +1,24 @@
-# Example Project: Multiple deployments for session-based games
+# Multiple deployments for session-based games
+> **This tutorial uses the Example Project from the GDK's [Set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro)**</br>
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro.md]:
```suggestion
> **This tutorial uses the Example Project from the GDK's [setup guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro)**</br>
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro.md [hunk 6]
```diff
@@ -1,21 +1,24 @@
...
+
 
 In this tutorial, you will use the [Deployment Manager](https://github.com/spatialos/deployment-manager) to launch multiple deployments of the [Example Project]({{urlRoot}}/content/get-started/example-project/exampleproject-intro) in the cloud.</br>
 
+Multiple deployments of your game are useful for time-limited match-based games where you have multiple game sessions with a defined number of players, such as battle royales.  The tutorial guides you through cloning the SpatialOS Deployment Manager repository and configuring the Example Project to work with the Deployment Manager, before launching your deployments via the Deployment Manager.
 
-Multiple deployments of your game are useful for time-limited match-based games where you have multiple game sessions with a defined number of players, such as battle royales.</br>
+**NOTES:**
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro.md]:
```suggestion
**Notes:** 
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro.md [hunk 7]
```diff
@@ -1,21 +1,24 @@
...
 
+Multiple deployments of your game are useful for time-limited match-based games where you have multiple game sessions with a defined number of players, such as battle royales.  The tutorial guides you through cloning the SpatialOS Deployment Manager repository and configuring the Example Project to work with the Deployment Manager, before launching your deployments via the Deployment Manager.
 
-Multiple deployments of your game are useful for time-limited match-based games where you have multiple game sessions with a defined number of players, such as battle royales.</br>
+**NOTES:** 
 
-**NOTE:** You can only use the Deployment Manager in the cloud; it isn't available for local deployments.
+* You can only use the Deployment Manager in the cloud; it isn't available for local deployments.
+* Before starting this tutorial you need to have completed the [Example Project set up guide.]({{urlRoot}}/content/get-started/example-project/exampleproject-intro)
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro.md]:
```suggestion
* Before starting this tutorial you need to complete the [Example Project set up guide.]({{urlRoot}}/content/get-started/example-project/exampleproject-intro) 
```

(this is Eddie talking in my head, saying "_need to have completed_ would be hard to translate!")

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-launch.md [hunk 8]
```diff
@@ -14,7 +15,7 @@ Where:
 
 * `<launch config path>` is the file path to the` one_worker_test.json` file in the Example project
 * `<snapshot path>` is the path to the snapshot file you generated in the [Example Project set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-local-deployment)
-*  `<assembly name>` is a name that you choose for your Deployment Manager assembly. This must be a different name to the assembly you created in [Step 4]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers#step-4-upload-your-worker-assemblies).
+*  `<assembly name>` is a name that you choose for your Deployment Manager assembly. This must be a different name to the assembly you created when you uploaded your worker assembly in [section 3. Build and upload your workers]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers#tutorial-deploymentmgr-workers#step-2-upload-your-worker-assemblies).
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-launch.md]:
```suggestion
*  `<assembly name>` is a name that you choose for your Deployment Manager assembly. This must be a different name to the assembly you created when you uploaded your worker assembly in [Steps 3 & 4: Build and upload your workers]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers#tutorial-deploymentmgr-workers#step-2-upload-your-worker-assemblies).
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md [hunk 9]
```diff
@@ -1,21 +1,19 @@
...
-
-The Launcher automatically downloads a game client for this deployment and runs it on your local machine.
-
-Once the game client has launched, you should see the Example Project start screen
-
-Select **QUICK JOIN** to join one of your sessions or select **BROWSE...** to choose from a list of currently running sessions. 
-
-The default game controls are listed below:
-
+2. select **LAUNCH** on the left of the page.<br/></br>
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md]:
```suggestion
2. Select **LAUNCH** on the left of the page.<br/></br>
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md [hunk 10]
```diff
@@ -1,21 +1,19 @@
...
+![img]({{assetRoot}}assets/deployment-manager/deploymentmgr-consoleoverview.png)
+_Image: The SpatialOS Console with the game client **LAUNCH** button highlighted._<br/></br>
+3. select the **Launch** button that appears in the center of the page to open the [SpatialOS Launcher](https://docs.improbable.io/reference/13.7/shared/operate/launcher). The Launcher automatically downloads a game client for this deployment and runs it on your local machine. </br></br>
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md]:
```suggestion
3. Select the **Launch** button that appears in the center of the page to open the [SpatialOS Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher). The Launcher automatically downloads a game client for this deployment and runs it on your local machine. </br></br>
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md [hunk 11]
```diff
@@ -1,21 +1,19 @@
-## Step 7: Play your game
+# Multiple deployments for session-based games
+## Step 6: Play your game
 
 1.  select **session_0** from the deployment list. This opens the deployment overview screen.
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md]:
```suggestion
1.  Select **session_0** from the deployment list. This opens the deployment overview screen.
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md [hunk 12]
```diff
@@ -1,21 +1,19 @@
...
-
-The default game controls are listed below:
-
+2. select **LAUNCH** on the left of the page.<br/></br>
+![img]({{assetRoot}}assets/deployment-manager/deploymentmgr-consoleoverview.png)
+_Image: The SpatialOS Console with the game client **LAUNCH** button highlighted._<br/></br>
+3. select the **Launch** button that appears in the center of the page to open the [SpatialOS Launcher](https://docs.improbable.io/reference/13.7/shared/operate/launcher). The Launcher automatically downloads a game client for this deployment and runs it on your local machine. </br></br>
+![img]({{assetRoot}}assets/deployment-manager/deploymentmgr-launch.png)
+_Image: The SpatialOS Console **Launch a client** pop-up window._<br/></br>
+4. Once the game client has launched, you see the Example Project start screen.</br>
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md]:
```suggestion
1. Once the game client has launched, you see the Example Project start screen.</br>
```

File: SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md [hunk 13]
```diff
@@ -67,11 +67,13 @@ When your deployment has launched, SpatialOS automatically opens the [Console](h
 When you’re done shooting your friends, you can click the **Stop** button in the [Console](https://console.improbable.io) to stop your deployment.
 
 <br/>
-### Next steps
-We hope you've enjoyed this tutorial. If you want to build a new game using the SpatialOS GDK, you should build it on top of the [SpatialOS GDK Starter template]({{urlRoot}}/content/get-started/gdk-template). If you want to port your existing game to SpatialOS, follow the [porting guide]({{urlRoot}}/content/tutorials/tutorial-porting-guide).
+** Next steps:**
+
+* If you want to create a new game using the SpatialOS GDK, you should get the [Starter Template]({{urlRoot}}/content/get-started/gdk-template).
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md]:
```suggestion
* If you want to create a new game using the GDK, you should get the [Starter Template]({{urlRoot}}/content/get-started/gdk-template). 
```

File: SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md [hunk 14]
```diff
@@ -67,11 +67,13 @@ When your deployment has launched, SpatialOS automatically opens the [Console](h
 When you’re done shooting your friends, you can click the **Stop** button in the [Console](https://console.improbable.io) to stop your deployment.
 
 <br/>
-### Next steps
-We hope you've enjoyed this tutorial. If you want to build a new game using the SpatialOS GDK, you should build it on top of the [SpatialOS GDK Starter template]({{urlRoot}}/content/get-started/gdk-template). If you want to port your existing game to SpatialOS, follow the [porting guide]({{urlRoot}}/content/tutorials/tutorial-porting-guide).
+** Next steps:**
+
+* If you want to create a new game using the SpatialOS GDK, you should get the [Starter Template]({{urlRoot}}/content/get-started/gdk-template). 
+* If you want to port your existing game to SpatialOS, follow the [Porting guide]({{urlRoot}}/content/tutorials/tutorial-porting-guide).
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md]:
```suggestion
* If you want to port your existing game to the GDK, follow the [Porting guide]({{urlRoot}}/content/tutorials/tutorial-porting-guide).
```

File: SpatialGDK/Documentation/content/tutorials/tutorial-porting-guide.md [hunk 15]
```diff
@@ -220,13 +222,16 @@ You can change:
 * the snapshot file's filename and location
 * the launch configuration
 
-## Next steps
+**Next steps:** </nr>
+If you haven't already, check out:
+
+* The tutorial on [multiple deployment for session-based games]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro).
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/tutorial-porting-guide.md]:
```suggestion
* The tutorial on [multiple deployments for session-based games]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro).
```

File: SpatialGDK/Documentation/content/tutorials/tutorial-porting-guide.md [hunk 16]
```diff
@@ -220,13 +222,16 @@ You can change:
...
+*  The Multiserver shooter tutorial tutorial to learn how to implement [cross-server interactions]({{urlRoot}}/content/tutorials/multiserver-shooter/tutorial-multiserver-intro).
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/content/tutorials/tutorial-porting-guide.md]:
```suggestion
*  The Multiserver Shooter tutorial to learn how to implement [cross-server interactions]({{urlRoot}}/content/tutorials/multiserver-shooter/tutorial-multiserver-intro).  
```

File: SpatialGDK/Documentation/known-issues.md [hunk 17]
```diff
@@ -7,7 +7,8 @@ For all other known issues, please visit [this board](https://github.com/spatial
 
 <br/>
 
-------------
-*2019-04-25 Page updated with full editorial review </br>
-2019-04-25 Removed out-dated known issues from this page and added links to current known issues.*
+<br/>--------<br/>
+
+*2019-04-25 Page updated with editorial review* </br>
+*2019-04-25 Removed out-dated known issues from this page and added links to current known issues.*
```
Comment (anne-edwards) [inline SpatialGDK/Documentation/known-issues.md]:
```suggestion
*2019-04-25 Removed outdated known issues from this page and added links to current known issues.*
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-configure.md [hunk 18]
```diff
@@ -72,7 +74,7 @@ In the `config.json`, you need to change the following lines:
 
 - `"ClientType": "YourClient"`, to `"ClientType": "UnrealClient"`,
 - `"NumberOfDeployments": 1`, to `"NumberOfDeployments": 2`,
-- `"AssemblyName": "YourAssembly"`, to `"AssemblyName": "<assembly_name>)"`, where `<assembly_name>` is the name you gave to your assembly in [Step 4]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers#step-4-upload-your-worker-assemblies) 
+- `"AssemblyName": "YourAssembly"`, to `"AssemblyName": "<assembly_name>)"`, where `<assembly_name>` is the name you gave to your assembly when you [uploaded your worker assemblies]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers#step-2-upload-your-worker-assemblies)
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-configure.md]:
```suggestion
- `"AssemblyName": "YourAssembly"`, to `"AssemblyName": "<assembly_name>)"`, where `<assembly_name>` is the name you gave to your assembly when you [uploaded your worker assemblies]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers#step-2-upload-your-worker-assemblies).
```

File: SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md [hunk 19]
```diff
@@ -1,21 +1,19 @@
...
-Select **QUICK JOIN** to join one of your sessions or select **BROWSE...** to choose from a list of currently running sessions. 
-
-The default game controls are listed below:
-
+# Multiple deployments for session-based games
+## Step 6: Play your game
+
+1.  Select **session_0** from the deployment list. This opens the deployment overview screen.
+1. Select **LAUNCH** on the left of the page.<br/></br>
+![img]({{assetRoot}}assets/deployment-manager/deploymentmgr-consoleoverview.png)
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-play.md]:
```suggestion
![img]({{assetRoot}}assets/deployment-manager/deploymentmgr-consoleoverview.png)</br>
```

**Pull request merged**